handlers/api_eth1.go

```diff
@@ -955,19 +955,21 @@ func getValidatorExecutionPerformance(queryIndices []uint64) ([]types.ExecutionP
 	err = db.ReaderDb.Select(&performanceList, `
 		SELECT 
 		validatorindex,
-		CAST(COALESCE(mev_performance_365d, 0) AS bigint) AS el_performance_365d,
-		CAST(COALESCE(mev_performance_total, 0) AS bigint) AS el_performance_total
+		CAST(COALESCE(mev_performance_365d, 0) AS text) AS el_performance_365d,
+		CAST(COALESCE(mev_performance_total, 0) AS text) AS el_performance_total
 		FROM validator_performance WHERE validatorindex = ANY($1)`, validatorsPQArray)
 	if err != nil {
 		return nil, fmt.Errorf("error can cl performance from db: %w", err)
 	}
 	for _, val := range performanceList {
+		performance365d, _ := big.NewInt(0).SetString(val.Performance365d, 10)
+		performanceTotal, _ := big.NewInt(0).SetString(val.PerformanceTotal, 10)
```

Comment:
There is no need to set the value to 0 first, you can use
```suggestion
		performance365d, _ := new(big.Int).SetString(val.Performance365d, 10)
		performanceTotal, _ := new(big.Int).SetString(val.PerformanceTotal, 10)
```